From e41bc69bcbf880196212524803e96521f5aa7c66 Mon Sep 17 00:00:00 2001
From: Scott Little <scott.little@windriver.com>
Date: Mon, 2 Oct 2017 17:53:00 -0400
Subject: [PATCH 07/10] WRS:
 0008-Add-patch-for-journald-config-rate-limit.patch

Conflicts:
	SPECS/systemd.spec
---
 SPECS/systemd.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/SPECS/systemd.spec b/SPECS/systemd.spec
index 48ac1bb..332514c 100644
--- a/SPECS/systemd.spec
+++ b/SPECS/systemd.spec
@@ -542,6 +542,7 @@ Patch0499: 0499-Revert-udev-net_id-add-support-for-phys_port_name-at.patch
 #WRS Patches
 Patch0501: 0501-inject-millisec-in-syslog-date.patch
 Patch0503: 0503-Configure-journald-to-forward-to-syslog.patch
+Patch0504: 0504-Configure-journald-rate-limit.patch
 
 %global num_patches %{lua: c=0; for i,p in ipairs(patches) do c=c+1; end; print(c);}
 
-- 
1.9.1

